testing/provider_client_test.go

```diff
@@ -201,8 +205,8 @@ func TestReauthEndLoop(t *testing.T) {
 	wg.Wait()
 	th.AssertEquals(t, info.reauthAttempts, 6)
 	th.AssertEquals(t, info.maxReauthReached, true)
-	th.AssertEquals(t, errAfter, 6)
-	th.AssertEquals(t, errUnable, 14)
+	th.AssertEquals(t, errAfter > 5, true)
+	th.AssertEquals(t, errUnable < 15, true)
```

Comment:
I don't think it's possible to accurately get an exact number here. I looked at the other concurrency tests and there's always a small range of valid values, so I think checking for a range of values is good enough.